closed/get_j9_source.sh

```diff
@@ -26,7 +26,7 @@
 set -e
 
 usage() {
-	echo "Usage: $0 [-h|--help] [-openj9-repo=<j9vm repo url>] [-openj9-branch=<branch>] [-openj9-sha=<commit sha>] [... other OpenJ9 repositories and branches options] [-parallel=<true|false>]"
+	echo "Usage: $0 [-h|--help] [-openj9-repo=<j9vm repo url>] [-openj9-branch=<branch>] [-openj9-sha=<commit sha>] [... other OpenJ9 repositories and branches options] [-vendor-source=<url>,<branch>,<sha>] [-parallel=<true|false>]"
```

Comment:
It should be `[-vendor-source=<url>,<branch>[,<sha>]]` since the `<sha>` is optional

Reply:
fixed